packages/server/src/api/controllers/row/utils/basic.ts

```diff
@@ -108,6 +117,11 @@ export function basicProcessing({
       thisRow[fieldName] = value
     }
   }
+
+  for (let aggregation of aggregations || []) {
+    thisRow[aggregation.name] = row[aggregation.name]
+  }
```

Comment:
Almost everything in this PR is in service of a few bits of code that look like this. We plumb `aggregations` all the way to the bottom of the SQL layer in order to slightly change the SQL query, and then make sure those fields get pulled out and don't get left behind by the several places that make sure _only_ schema fields get pushed back out through the API. It feels very messy to me.